content/influxdb/v2.0/backup-restore/backup.md

```diff
@@ -18,22 +18,6 @@ data and metadata stored in InfluxDB.
 InfluxDB copies all data and metadata to a set of files stored in a specified directory
 on your local filesystem.
 
-{{% warn %}}
-#### InfluxDB 2.0rc (release candidate)
-
-The `influx backup` command is not compatible with InfluxDB 2.0rc (release candidate).
-To back up data,
-
-1. Stop `influxd`.
-2. Manually copy the InfluxDB data directories:
-
-   ```
-   cp -r ~/.influxdbv2 ~/.influxdbv2_bak
-   ```
-
-For more information, see [Upgrade to InfluxDB OSS 2.0rc](/influxdb/v2.0/reference/upgrading/rc-upgrade-guide/).
-{{% /warn %}}
-
```

Comment:
I'm somewhat torn about removing this section. It's still relevant to anybody who is still using a beta version. But I would hope everyone has upgraded to an RC (says the guy who just did it this week). I think we probably need to leave this in for now.

Reply:
Got it. added back.